code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -335,8 +335,8 @@
 	icon_closed = "brown_jacket_nt"
 
 /obj/item/clothing/suit/storage/toggle/agent_jacket
```

Comment:
check if we have another already existing FBI jacket, if so just delete this

Reply:
we do have another so its tossed